braindecode/datautil/preprocess.py

```diff
@@ -251,3 +253,63 @@ def scale(data, factor):
     if hasattr(data, '_data'):
         data._data = scaled
     return scaled
+
+
+def filterbank(raw, frequency_bands, drop_original_signals=True, n_jobs=1,
+               method='fir', iir_params=None, phase='zero', fir_window='hamming',
+               fir_design='firwin', **filter_kwargs):
```

Comment:
why not simply having **filter_kwargs and say it supports params of filter function from mne?

if feels you explicit some params and some not.

Reply:
In another PR you mentioned disliking kwargs and to make arguments explicit. I therefore took the arguments that are relevant for my use case and made them explicit, while at the same time keeping full compatibility with mne through filter_kwargs. I will now update it again to be less explicit and only have the filter_kwargs. The docstring for this parameter will then contain a referral to mne.

Reply:
arfff to be it's one option or the other. If it was me and I would copy all
the mne parameters and copy the docstring. After updating the docstring
5 times due to mne changes I would reconsider. Note that mne release
are every 6 months so...